config.go

```diff
@@ -460,9 +461,17 @@ func LoadConfig(filename string, notExistErr bool) int {
 
 	configLastMod[filename] = tmpConfigLastMod
 
-	if err := json.Unmarshal(jsonc.ToJSON(configFile), &config); err != nil {
-		Log("LoadConfig", GetLangText("Error-ParseConfig"), true, filename, err.Error())
-		return -4
+	switch filepath.Ext(filename) {
```

Comment:
It is recommended to convert it to lowercase (strings.ToLower)

Reply:
Fixed.